src/Theme_Mod_Command.php

```diff
@@ -85,49 +85,113 @@ public function get( $args = array(), $assoc_args = array() ) {
 			$args = array();
 		}
 
-		$list = array();
-		$mods = get_theme_mods();
+		// This array will hold the list of theme mods in a format suitable for the WP CLI Formatter.
+		$mod_list = array();
+
+		// If specific mods are requested, filter out any that aren't requested.
+		$mods = ! empty( $args ) ? array_intersect_key( get_theme_mods(), array_flip( $args ) ) : get_theme_mods();
 		if ( ! is_array( $mods ) ) {
-			// If no mods are set (perhaps new theme), make sure foreach still works.
+			// If no mods are set (perhaps new theme), make sure array_intersect_key still works.
 			$mods = array();
 		}
-		foreach ( $mods as $k => $v ) {
-			// If mods were given, skip the others.
-			if ( ! empty( $args ) && ! in_array( $k, $args, true ) ) {
-				continue;
-			}
 
-			if ( is_array( $v ) ) {
-				$list[] = [
-					'key'   => $k,
-					'value' => '=>',
-				];
-				foreach ( $v as $_k => $_v ) {
-					$list[] = [
-						'key'   => "    $_k",
-						'value' => $_v,
-					];
-				}
-			} else {
-				$list[] = [
-					'key'   => $k,
-					'value' => $v,
-				];
+		// Generate the list of items ready for output. We use an initial separator that we can replace later depending on format.
+		$separator = '!!!';
+		array_walk(
+			$mods,
+			function ( $value, $key ) use ( &$mod_list, $separator ) {
+				$this->mod_to_string( $key, $value, $mod_list, $separator );
 			}
-		}
+		);
 
-		// For unset mods, show blank value.
-		foreach ( $args as $mod ) {
-			if ( ! isset( $mods[ $mod ] ) ) {
-				$list[] = [
-					'key'   => $mod,
-					'value' => '',
-				];
-			}
+		// Take our Formatter-friendly list and adjust it according to the requested format.
+		switch ( \WP_CLI\Utils\get_flag_value( $assoc_args, 'format' ) ) {
```

Comment:
This should be simplified by importing `WP_CLI\Utils` and then using `Utils\get_flag_value()` here instead.